src/Microsoft.Dnx.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -42,9 +43,15 @@ public NuGetDependencyResolver(PackageRepository repository)
 
         public void ApplyLockFile(LockFile lockFile)
         {
+            ApplyOptimizedLockFile(new OptimizedLockFile(lockFile));
+        }
+
+        public void ApplyOptimizedLockFile(OptimizedLockFile optimizedLockFile)
+        {
+            _compatibilityChecker = new CompatibilityChecker(optimizedLockFile);
```

Comment:
We cannot null `_compatibilityChecker` out because it will be needed in another member method `GetDescription`